bin/audioldm2

```diff
@@ -11,6 +11,15 @@ matplotlib_logger.setLevel(logging.WARNING)
 
 parser = argparse.ArgumentParser()
 
+parser.add_argument(
+    "--mode",
+    type=str,
+    required=False,
+    default="generation",
+    help="generation: text-to-audio generation; transfer: style transfer",
+    choices=["generation", "transfer", "sr_ip"]
```

Comment:
For different mode, e.g., "generation", "sr_ip", please also add a subsection in the readme to introduce them. You can refer to the codebase of AudioLDM.